packages/kbn-unified-data-table/src/components/data_table.tsx

```diff
@@ -140,7 +129,7 @@ export interface DiscoverGridProps {
   /**
    * Grid display settings persisted in Elasticsearch (e.g. column width)
    */
-  settings?: DiscoverGridSettings;
+  settings?: UnifiedDataTableSettings;
   /**
    * Saved search description
    */
```

Comment:
now we are no longer limited to Saved searches, so it could be `Search description` and `Search title` so it's more generic